scout/server/blueprints/variant/templates/variant/variant.html

```diff
@@ -475,36 +475,38 @@ <h6>No matching causative or managed variants</h6>
           </li>
         {% endif %}
       </ul>
-        <table class="table table-fixed table-sm mt-3">
-          <tbody>
-            <tr>
-              <td style="width:15%;">
-                {{ igv_button(case, variant, case_groups) }}
-              </td>
-              {% if case.mt_bams or case.bam_files %}
-              {% endif %}
-              {% if config.SQLALCHEMY_DATABASE_URI %}
-                {% for gene in variant.genes %}
-                  <td>
-                    <a target="_blank" href="{{ url_for('report.gene', gene_id=gene.hgnc_id, sample_id=variant.samples|map(attribute='sample_id')|list) }}">
-                      {{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }}
-                    </a>
-                  </td>
-                {% endfor %}
-              {% endif %}
-            </tr>
-            <tr>
-              <td>
-                {{ igv_track_selection(igv_tracks, current_user) }}
-              </td>
-            </tr>
-            {% if splice_junctions_tracks %}
-              <tr>
-                <td>{{ splice_junctions_button(institute._id, case.display_name, variant._id) }}</td>
-              <tr>
-            {% endif %}
-          </tbody>
-        </table>
+      <ul class="list-group">
+        <li class="list-group-item d-flex justify-content-between">
+          <div>
+            {{ igv_button(case, variant, case_groups) }}
+          </div>
+          
+          {% if case.mt_bams or case.bam_files %}
+          {% endif %}
```

Comment:
@northwestwitch @dnil Do we need this on line 484-485 and what does it do?

Reply:
Looks like some leftover code that I forgot to remove last time I touched this file. You can certainly remove these lines!